geode-management/src/main/java/org/apache/geode/management/configuration/ClassName.java

```diff
@@ -35,8 +36,8 @@
 public class ClassName implements Serializable {
```

Comment:
Is this class only intended to be serialized with the spring ObjectMapper? If so does it need to implement Serializable? I don't think we ever intend to use standard jdk serialization but since it is an external api and since it implements Serializable, we should either remove Serializable (and the uid) from it, or we should test it and make sure standard jdk serialization works with it.

Reply:
I believe this class is used to be sent from locator to server, that's where it needs to be Serializable.